.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
intentional to have this file?

Reply:
Opinions on having PyCharm project files in the repo, yay or nay?